CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## v1.0.0 (2023-10-26)
+
+* Support firebase/php-jwt: ^6.0
```

Comment:
```suggestion
* Upgrade to firebase/php-jwt: ^6.0 - in most cases this will not affect external users of this library. However if you are directly using the included `CertificateProvider` classes note that these now return an array of `Firebase\JWT\Key` objects, not strings.
```